### PR TITLE
changed emails in notes to mailto links

### PR DESCRIPTION
**Pull request by Kefaku**
replaced emails in `notes` with `<a href=\"\">mailto:</a>` links

### CODE REVIEW

**Comment by daviddavo** [comment]
Hm... I think it has too much html. Perhaps we should change the generator code so it uses Markdown instead?

I think `send then an [email](contact@example.com)` it's easier to read

**Comment by Kefaku** [comment]
> Hm... I think it has too much html. Perhaps we should change the generator code so it uses Markdown instead?
> 
> I think `send then an [email](contact@example.com)` it's easier to read
> 
> What do you think?

True, @daviddavo , but I don't know how to do that and currently I don't have enough time to find out.

**Comment by Kefaku** [comment]
updated with markdown, @daviddavo

File: _data/sites.json [hunk 1]
```diff
@@ -59,9 +59,9 @@
         "name": "Badoo",
         "url": "https://badoo.com/feedback",
         "difficulty": "hard",
-        "notes": "Go to the url provided above and submit a request, picking 'General question' as subject, or send an email to <a href=\"mailto:DPOcorp.badoo.com\">DPOcorp.badoo.com</a>.",
-        "notes_es-ES": "Entra en la dirección indicada arriba y envía una solicitud, selecionando como asunto 'Pregunta general', o envía un email a <a href=\"mailto:DPOcorp.badoo.com\">DPOcorp.badoo.com</a>.",
-        "notes_pl-PL": "Wejdź na powyższy adres i wyślij zapytanie, wybierając jako temat 'Zapytanie ogólne', lub wyślij e-mail na adres <a href=\"mailto:DPOcorp.badoo.com\">DPOcorp.badoo.com</a>."
+        "notes": "Go to the url provided above and submit a request, picking 'General question' as subject, or send an email to [DPOcorp.badoo.com](mailto:DPOcorp.badoo.com).",
```
Comment (daviddavo) [inline _data/sites.json:62]: That doesn't seem like an email... The correct email is DPO@corp.badoo.com (i guess the `@` went missing...

**Review by daviddavo — CHANGES_REQUESTED** [review]
Could you correct the badoo email before merging?

File: _data/sites.json (hunk 1 above)
Comment (Kefaku) [reply _data/sites.json]: done, @daviddavo

**Review by daviddavo — APPROVED** [review]

**Pull request merged**